features/cart/shopping_cart/adding_product_to_cart_with_prices_dependent_on_channel.feature

```diff
@@ -21,6 +21,13 @@ Feature: Adding a product to cart with prices dependent on a channel
         When I add product "Leprechaun's Gold" to the cart
         Then I should see "Leprechaun's Gold" with unit price "€10.00" in my cart
 
+    @api
+    Scenario: Buying a product in default currency for browsed channel
+        Given I change my current channel to "Web-EU"
+        When I pick up my cart
+        And I add product "Leprechaun's Gold" to the cart
+        Then I should see "Leprechaun's Gold" with unit price "€10.00" in my cart
+
```

Comment:
There is no need to add this scenario, you could add an `@api` tag to the previous scenario

Reply:
Ok but it doesn't work without `When I pick up my cart`

Reply:
you can pick up a cart in step `I add product "Leprechaun's Gold" to the cart` if cart does not exist :)